### PR TITLE
17831 - EFT Statement notification

### PR DESCRIPTION
**Pull request by ochiu**
*Issue #:*
https://github.com/bcgov/entity/issues/17831

*Description of changes:*
- add statement notification template (currently will be used only by EFT payment methods)
- add utility function to help with day suffixes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by ochiu** [comment]
Screenshot 2023-10-12 at 9 53 29 AM

File: queue_services/account-mailer/src/account_mailer/email_templates/statement_notification.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Statement Available
+
+Your monthly account statement for the period of {{ from_date }} to {{ to_date }} is now available.
+
+The total amount owed for your {{ account_name }} during this statement period is ${{ total_amount_owing }}
+
+[Log in to view your {{ statement_frequency }} statement]({{ payment_statement_url }})
```
Comment (ochiu) [inline queue_services/account-mailer/src/account_mailer/email_templates/statement_notification.html:7]: Missing contact information. Will get this added soon

File: queue_services/account-mailer/src/account_mailer/worker.py [hunk 2]
```diff
@@ -48,7 +48,8 @@
 from account_mailer.enums import Constants, MessageType, SubjectType, TemplateType, TitleType
 from account_mailer.services import minio_service  # pylint: disable=wrong-import-order
 from account_mailer.services import notification_service  # pylint: disable=wrong-import-order
-from account_mailer.utils import format_currency, get_local_formatted_date  # pylint: disable=wrong-import-order
+from account_mailer.utils import format_day_with_suffix  # pylint: disable=wrong-import-order
```
Comment (seeker25) [inline queue_services/account-mailer/src/account_mailer/worker.py:51]: any idea why we're using disable=wrong-import-order?

**Comment by seeker25** [comment]
Account mailer version update please

**Review by seeker25 — APPROVED** [review]

File: queue_services/account-mailer/src/account_mailer/worker.py (hunk 2 above)
Comment (ochiu) [reply queue_services/account-mailer/src/account_mailer/worker.py:51]: no idea, I probably copy and pasted it from another line but I will remove all of them and fix any linting issues

Comment (ochiu) [reply queue_services/account-mailer/src/account_mailer/worker.py]: removed the pylint hints and cleaned up the imports

**Pull request merged**